packages/cli/src/controller/codegen-controller.spec.ts

```diff
@@ -0,0 +1,66 @@
+// Copyright 2020-2021 OnFinality Limited authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+import fs from 'fs';
+import ejs from 'ejs';
+import {makeSchema, renderTemplate, generateSchema} from './codegen-controller';
+jest.mock('fs', () => {
+  const fsMock = jest.createMockFromModule('fs') as any;
+  fsMock.promises = {
+    writeFile: jest.fn(),
+    access: jest.fn(),
+  };
+  fsMock.readFileSync = jest.fn();
+  return fsMock;
+});
+
+jest.mock('ejs', () => {
+  const ejsMock = jest.createMockFromModule('ejs') as any;
+  ejsMock.renderFile = jest.fn();
```

Comment:
This line seems not necessary, it has been done by `createMockFromModule`

Reply:
removed